OptimizelySDKUniversal/OptimizelySDKUniversal.xcodeproj/project.pbxproj

```diff
@@ -1892,6 +1892,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ALWAYS_SEARCH_USER_PATHS = NO;
+				BITCODE_GENERATION_MODE = marker;
```

Comment:
shouldn't this be embed?

Reply:
This is for debug build which should use "fembed-bitcode-marker" flag while compiling.

Reply:
does, it automatically add embed because it isn't in the project file at all?

Reply:
Yes.